protocols/grpc/agent.proto

```diff
@@ -187,6 +187,8 @@ message UpdateRoutesRequest {
 }
 
 message OnlineCPUMemRequest {
+	// Specify the number of CPUs that were added
+	uint32 cpus = 1;
```

Comment:
Was thinking that `nb_cpus` or something with number in it, might be more appropriate as the parameter here, since we're providing a number of CPUs.